utils/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod config;
```

Comment:
Do you think it's worth having a separate `lib` and `config` file? Since there's only the one module we could just put the `mod config {...}` inside the `lib.rs` file.

Reply:
good point!